README.md

```diff
@@ -11,111 +16,148 @@
 
-## Can I use Baritone as a library in my custom utility client?
+***
 
-That's what it's for, sure! (As long as usage complies with the LGPL 3.0 License)
+*When will support for Minecraft 1.17.~ be released?*
 
-## How is it so fast?
+Join the [Baritone Discord Server](https://discord.com/invite/s6fRBAUpmr) to be notified of future releases.
 
-Magic. (Hours of [leijurv](https://github.com/leijurv/) enduring excruciating pain)
+***
 
-### Additional Special Thanks To:
+*Is using Baritone cheating?*
+
+No, Baritone is a utility mod intended to automate certain game mechanics.
+
+Do **NOT** use Baritone on public servers which explicitly ban utility clients and other mods which may give
+you an unfair advantage over other players.
+
+***
+
+*How do I use Baritone?*
+
+Read [Usage.md](USAGE.md) or 
+watch [Impact Client's YouTube playlist.](https://www.youtube.com/playlist?list=PLnwnJ1qsS7CoQl9Si-RTluuzCo_4Oulpa)
+
+***
+
+*How does Baritone work and how is it so fast?*
+
+~~Read the source code / documentation~~. 
+
+**Magic** (Over 2 years of [leijurv](https://github.com/leijurv/) enduring excruciating pain)
+
+***
+
+*Why is this project named "Baritone?"*
+
+Read: [Baritone](https://en.wikipedia.org/wiki/Baritone)
+
+Named for [FitMC's](https://www.youtube.com/user/SonOfShoop) deep, sultry voice.
+
+***
+
+### Special Thanks To:
 
 ![YourKit-Logo](https://www.yourkit.com/images/yklogo.png)
 
 YourKit supports open source projects with innovative and intelligent tools for monitoring and profiling Java and .NET applications.
 
-YourKit is the creator of the [YourKit Java Profiler](https://www.yourkit.com/java/profiler/), [YourKit .NET Profiler](https://www.yourkit.com/.net/profiler/), and [YourKit YouMonitor](https://www.yourkit.com/youmonitor/).
+YourKit is the creator of the [YourKit Java Profiler](https://www.yourkit.com/java/profiler/),
+[YourKit .NET Profiler](https://www.yourkit.com/.net/profiler/)
+and [YourKit YouMonitor](https://www.yourkit.com/youmonitor/).
+
+We thank them for granting Baritone an OSS license so that we can ensure our software is the best it can be.
+
+### Extras
+
+There was at least 1 commit per day from August 2018 - August 2019 (More hours of [leijurv](https://github.com/leijurv/) enduring excruciating pain)
 
-We thank them for granting Baritone an OSS license so that we can make our software the best it can be.
+<p align="center">
+  <a href="http://forthebadge.com/"><img src="https://forthebadge.com/images/badges/built-with-swag.svg" alt="forthebadge"/></a>
+  <a href="http://forthebadge.com/"><img src="https://forthebadge.com/images/badges/mom-made-pizza-rolls.svg" alt="forthebadge"/></a>
+</p>
+
+### Stars over time
 
-## Why is it called Baritone?
+The exponential increase in stars can be attributed to the namesake of this project,
+[FitMC](https://www.youtube.com/user/SonOfShoop) who posted 2 videos about Baritone on August 2019 - September 2019.
```

Comment:
Again, is this necessary, do we need to talk about fitmc posting videos about it?